docs/reference/mapping/runtime.asciidoc

```diff
@@ -752,6 +761,79 @@ The API returns the following result. Without building your data structure in
 advance, you can search and explore your data in meaningful ways to experiment
 and determine which fields to index.
 
+Also, remember that `if` statement in the script?
+
+[source,txt]
```

Comment:
I think we were using `[source,painless]` for this. It probably doesn't make a big difference though.

Reply:
I'll update it 👍